docs/reference/apis.md

```diff
@@ -89,3 +89,9 @@
 ::: meadowrun.AwsSecret
 
 ::: meadowrun.AzureSecret
+
+## Specifying resource thresholds
+
+::: meadowrun.Threshold
+::: meadowrun.Resources
```

Comment:
We already have Resources and AllocEC2Instance above? Maybe we could just link to where they already are? Or maybe merge with the `Specifying resource requirements` section? I think it would be confusing to duplicate classes/functions here.